src/blocks/net.rs

```diff
@@ -386,6 +391,10 @@ impl NetConfig {
         true
     }
 
+    fn default_bits() -> bool {
+        true
```

Comment:
We probably don't want this to be true by default, since that would break existing configurations.